### ISSUE TITLE
`javascript.express.security.injection.tainted-sql-string.tainted-sql-string` causes false positives

### ISSUE THREAD

**Issue by ianmuscat** [returntocorp/sgrep-rules#2575]
**Describe the bug**
The `javascript.express.security.injection.tainted-sql-string.tainted-sql-string` causes false positives and seems to trigger despite express not being imported, or SQL statements not being used (or containing user input).

**To Reproduce**
```javascript
function doStuff(sequelize, bar) {
    let bing = sequelize.query('SELECT count(foo) AS count FROM bar WHERE id = 3')
    console.log(`select ${bing.count}`);
    return;
}
```
```javascript
function doStuff(foo, bar) {
    let bing = foo.headers('')
    console.log(`drop ${bing}`);
    return;
}
```

**Expected behavior**
I would expect this rule to trigger only when express is being used and the input from an HTTP request is being used in a Sequelize query without the use of `replacements`. 

**Priority**
How important is this to you?
- **P2**: *annoying but not blocking me*

**Comment by LewisArdern** [comment]
@ianmuscat this rule was updated to only come from request sources, the example you showed does not trigger, if this behavior is still something which happens let me know
